### PR TITLE
Issue/4548 Connect card reader UI

### LINKED ISSUE #4548: Connect Card Reader screen - UI updates
Hi there, 

Can we make the following alignment updates to the Connect card reader screen? Thanks!
Screen Shot 2021-07-06 at 9 47 57 AM

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #4548

### Description

The PR:
* Changes the UI of "search" reader screen to follow [the design](vaw3DvewbUwuQnPXpdNGGH-fi-38%3A7368)
* UI is implemented with swiftUI
* Small fixes to the UI
* Reuses "learn more" section 

### Testing instructions

* Open settings
* Click in-person payments
* Click Manage card readers
* Notice the improved UI

### Screenshots

https://user-images.githubusercontent.com/4923871/148056653-7619e587-4d4f-4d4e-b578-82fbea3da2c8.mov

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ctarda** [comment]
@allendav you know this part of the system better than me, would you mind reviewing please?

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift [hunk 1]
```diff
@@ -112,293 +96,144 @@ private extension CardReaderSettingsSearchingViewController {
...
-        return UITableView.automaticDimension
-    }
-
-    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let row = rowAtIndexPath(indexPath)
-        let cell = tableView.dequeueReusableCell(withIdentifier: row.reuseIdentifier, for: indexPath)
-        configure(cell, for: row, at: indexPath)
-
-        return cell
+    var isCompat: Bool {
```
Comment (allendav) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift]: Maybe `isCompact` ?

**Review by allendav — APPROVED** [review]
Before (iPad):



After:

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift (hunk 1 above)
Comment (kidinov) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift]: Fixed - https://github.com/woocommerce/woocommerce-ios/pull/5612/commits/3cd74782a289b15ca567fb1a4955bae9b5f60836

**Pull request merged**